tests/clvm/test_curry_and_treehash.py

```diff
@@ -0,0 +1,21 @@
+from chia.types.blockchain_format.program import Program
+from chia.wallet.puzzles import p2_delegated_puzzle_or_hidden_puzzle  # import (puzzle_for_pk, puzzle_hash_for_pk, MOD)
+from chia.wallet.util.curry_and_treehash import calculate_hash_of_quoted_mod_hash, curry_and_treehash
+
+
+def test_curry_and_treehash() -> None:
+    arbitrary_mod = p2_delegated_puzzle_or_hidden_puzzle.MOD
+    arbitrary_mod_hash = arbitrary_mod.get_tree_hash()
+
+    # we don't really care what `arbitrary_mod` is. We just need some code
+
+    quoted_mod_hash = calculate_hash_of_quoted_mod_hash(arbitrary_mod_hash)
+
+    for v in range(500):
```

Comment:
Shouldn't we test edgecases like large strings and negatives rather than just positive integers 500 times?